ui/ui-components/components/enterpriseLink.tsx

```diff
@@ -1,12 +1,13 @@
 import Link from "next/link";
+import { grouparooUiEdition } from "../utils/uiEdition";
 
 const EnterpriseLink = Link;
 
 const CommunityLink = function ({ children }) {
   return <span>{children}</span>;
 };
 
-export default process.env.GROUPAROO_UI_EDITION === "enterprise" ||
-process.env.GROUPAROO_UI_EDITION === "config"
+export default grouparooUiEdition() === "enterprise" ||
+grouparooUiEdition() === "config"
```

Comment:
This pattern is strange to look at. It also feels a little odd that `<EnterpriseLink />` can actually be `<CommunityLink />`. Maybe this could be `<GrouparooLink />` or something?